third_party/sycl/crosstool/computecpp.tpl

```diff
@@ -78,111 +66,19 @@ def get_device_compiler_flags(compiler_flags):
   ]
   return compiler_flags + computecpp_flags
 
-def remove_unknown_computecpp_flags(flags):
-  """Remove flags not recognised by the device compiler. Also remove any CPU
-  specific vectorization flags as they are not applicable to SYCL devices."""
-  unknown_flags = [
-      '-fsanitize',
-      '-fno-canonical-system-headers',
-      '=native',
-      '=core2',
-      'msse',
-      'mavx',
-      'mmmx',
-      'm3dnow',
-      'fma'
-  ]
-  return [flag for flag in flags if not any(x in flag.lower() for x in unknown_flags)]
-
-def get_device_compiler_flags_legacy(compiler_flags):
-  """Remove any flags not used by the device compiler and set up the device
-  compiler flags and includes."""
-  computecpp_flags = [
-      '-sycl-compress-name',
-      '-Wno-unused-variable',
-      '-Wno-c++11-narrowing',
-      '-I', COMPUTECPP_INCLUDE,
-      '-isystem', COMPUTECPP_INCLUDE,
-      '-std=c++11',
-      '-sycl', '-emit-llvm', '-no-serial-memop',
-      '-Xclang', '-cl-denorms-are-zero',
-      '-Xclang', '-cl-fp32-correctly-rounded-divide-sqrt',
-      '-Xclang', '-cl-mad-enable',
-      '-mllvm', '-inline-threshold=10000000',
-  ]
-  return computecpp_flags + remove_unknown_computecpp_flags(compiler_flags)
-
-def add_sycl_env_vars_to_flags_legacy(flags):
-  """Add required env vars to compiler flags as needed for ComputeCpp."""
-  extra_env_vars = [
-      '-DTENSORFLOW_USE_SYCL=1',
-      '-DEIGEN_USE_SYCL=1',
-  ]
-
-  # Silence Clang warning
-  if "clang" in CPU_CXX_COMPILER:
-    extra_env_vars += [
-    '-Wno-unused-const-variable',
-    '-Wno-unused-command-line-argument'
-    ]
-
-  return flags + extra_env_vars
-
-def get_host_compiler_flags_legacy(compiler_flags, bc_out):
-  """Remove device compiler specific flags and set up the host compiler flags."""
-  device_flag_prefixes = ('-MF', '-MD',)
-  host_compiler_flags = [flag for flag in compiler_flags if (not flag.startswith(device_flag_prefixes) and not '.d' in flag)]
-  host_compiler_flags[host_compiler_flags.index('-c')] = "--include"
-  host_compiler_flags = [
-      '-DEIGEN_HAS_C99_MATH',
-      '-xc++',
-      '-Wno-unused-variable',
-      '-I', COMPUTECPP_INCLUDE,
-      '-c', bc_out
-  ] + host_compiler_flags
-  return host_compiler_flags
-
-def isDriverSupported():
+def checkComputeCppIsSupported():
   outputList = check_output([COMPUTECPP_DRIVER, '--version']).split(" ")
   ccpp_version_idx = outputList.index('Device') - 1
-
-  cppVersionList = outputList[ccpp_version_idx].split(".")
-
-  if((int(cppVersionList[0]) == 0 and int(cppVersionList[1]) >= 5) or int(cppVersionList[0]) > 0):
```

Comment:
Would it be better to keep this more complete version check instead of the one you use below?
(`if int(cppVersionList[0]) == 0 and int(cppVersionList[1]) < 5`)

Reply:
It's not really more complete. The part after the or is to check if we ever have a version that looks like 1.x.x but I don't need that since I'm only checking for a minimum requirement.

Reply:
Yes, you are right.Your check is the inverse, so it doesn't matter if the major version is higher.